packages/xo-server/src/fatfs-buffer.mjs

```diff
@@ -23,6 +23,28 @@ const SECTOR_SIZE = 512
 
 const TEN_MIB = 10 * 1024 * 1024
 
+// https://en.wikipedia.org/wiki/Master_boot_record
+// we'll add a classical generic mbr
+// with one FAT16 partition addressed by LBA
+
+export function addMbr(buf) {
+  const mbr = Buffer.alloc(SECTOR_SIZE, 0)
+  // 0 - 446 is bootstrap code , keep it empty
+
+  // entry
+  mbr[446] = 0x08 // entry is bootable
+  mbr[450] = 0x04 // MBR bootable , less than 32MB
+  mbr[454] = 1 // LBA address of first sector
+  mbr[455] = TEN_MIB / SECTOR_SIZE // LBA address of last sector
```

Comment:
Wouldn't it be better to use `buf.length` instead of `TEN_MIB`?

And add an `assert` to make sure it's an integer.